config/environments/production.rb

```diff
@@ -2,7 +2,11 @@
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
+  env_file = File.join(Rails.root, 'config', 'application.yml')
+  YAML.load_file(env_file, aliases: true).symbolize_keys[:development].each do |key, value|
```

Comment:
what is this @htoo-eain-lwin the `application.yml` shouldn't be used like this on production

Reply:
This is for bypassing Figaro [issue](https://github.com/laserlemon/figaro/issues/289). I will change to `symbolize_keys[development]` to `symbolize_keys[:production]` when we really want to use actual values like mailer host, mailer sender and etc. 
Since, the one we deployed it on render, doesn't really use these values,  therefore I just dump these.

Reply:
I mean we shouldn't load this on the `production` environment, not still sure the purpose here @htoo-eain-lwin do you have any issue without this changes 🤔

Reply:
The values from applicaiton.yml ```  MAILER_DEFAULT_HOST: "localhost"
  MAILER_DEFAULT_PORT: "3000"
  MAILER_SENDER: "Test <noreply@nimblehq.co>"``` need to be loaded because we are using ```config.action_mailer.asset_host = ENV.fetch('MAILER_DEFAULT_HOST')

  config.action_mailer.default_url_options = {
    host: ENV.fetch('MAILER_DEFAULT_HOST'),
    port: ENV.fetch('MAILER_DEFAULT_PORT')
  }``` in `confgi/environments/production.rb`.  Now I remove the code block and just put as environment variables on render . :D

Reply:
Adding the ENV on Render is the right move @htoo-eain-lwin